podcasts/Sharing/ShareDestination.swift

```diff
@@ -255,18 +255,19 @@ extension ShareDestination {
         }
 
         func exportVideo() async throws -> URL {
+            let scale: CGFloat = 3
```

Comment:
Can this be defined as Constant? I'm asking because we can read it from within SharingModal as the value is the same?